samples/test/samples.test.js

```diff
@@ -55,7 +69,7 @@ describe('samples', () => {
   describe('start-up script', () => {
     const name = `gcloud-apache-${uuid.v4().split('-')[0]}`;
     after(async () => deleteVM(name));
-    it('should create vm with startup script', function (done) {
```

Comment:
Looks like we need an `npm run fix`.  I'm curious - which editor do you use?  I'm curious how this happened.

Reply:
I'm using VS code. I'm also confused, my linting seems to be off!

Reply:
We have a feature request in `gts` to support a `.editorconfig` file type - I wonder if vscode would respect that by default 🤷 I'll play around with it